homeassistant/components/homematicip_cloud/device.py

```diff
@@ -84,10 +87,21 @@ def unique_id(self) -> str:
     @property
     def icon(self) -> Optional[str]:
         """Return the icon."""
+
         if hasattr(self._device, "lowBat") and self._device.lowBat:
             return "mdi:battery-outline"
         if hasattr(self._device, "sabotage") and self._device.sabotage:
             return "mdi:alert"
+        if hasattr(self._device, "deviceOverheated") and self._device.deviceOverheated:
```

Comment:
To allow future scaling I would recommend something like this:
```py
DEVICE_ATTRIBUTE_ICONS = {
    "deviceOverheated": "mdi:alert"
    ...
}
...
@property
def icon(self):
    for attr, icon in DEVICE_ATTRIBUTE_ICONS.items():
        if getattr(self._device, attr, None):
            return icon

    return None
```
The dict should be assumed to be ordered.

Reply:
Sounds good. Done